src/mono/wasm/test-main.js

```diff
@@ -336,7 +342,10 @@ async function run() {
 
         // this is subsequent run with the actual tests. It will use whatever was cached in the previous run. 
         // This way, we are testing that the cached version works.
-        mono_exit = exit;
+        mono_exit = (code, reason) => {
```

Comment:
Why we are doing this ?

Reply:
For debugging. If you run chrome headless, it closes after the tests finish. The `--no-exit` allows to leave the browser opened and iterate manually.

Reply:
I see this prevents "WASM EXIT" message.
Let's not complicate this more than it already is. 
We already have lot of trouble related to exit, quit, ExitStatus, unwind, abort and log flushing.

I guess you use `/p:WasmXHarnessArgs="--no-headless"`
I suggest that we add something similar to ignore the "WASM EXIT" on the other side and don't kill the browser process.